src/gcinfo/gcinfoencoder.cpp

```diff
@@ -1217,7 +1217,7 @@ void GcInfoEncoder::Build()
         // (after, of course, adding the size of the call instruction to get the return PC).
         callSite += m_pCallSiteSizes[callSiteIndex] - 1;
 
-        _ASSERTE(DENORMALIZE_CODE_OFFSET(NORMALIZE_CODE_OFFSET(callSite)) == callSite);
+        _ASSERTE(DENORMALIZE_CODE_OFFSET(NORMALIZE_CODE_OFFSET(callSite + 1)) == (callSite + 1));
```

Comment:
@jkotas I think that JIT passed the value as intended. At line 1218, there is an additional subtraction. It seems that callSite at line 1220 represents the valid offset - 1 value, and both ``NORMALIZE_CODE_OFFEST`` and ``DENORMALIZE_CODE_OFFSET`` require the valid offset as its argument.